### PR TITLE
feat: fix layout of homepage about

### PR DESCRIPTION
**Pull request by zalborzi**
## Summary by Sourcery

This pull request enhances the layout of the homepage 'About' section by adding a background image and updating the styling for better visual appeal and readability.

- **Enhancements**:
    - Updated the layout of the homepage 'About' section to include a background image and improved styling.

### CODE REVIEW

**Review by sulejmank — APPROVED** [review]

File: src/public/bg-background.png
Comment (brunopacheco1) [inline src/public/bg-background.png:1]: important: this image it too big, please optime the size, and remove `gdi-visuals.png` and its license file from public, as it is not used anymore.

File: package.json [hunk 1]
```diff
@@ -21,6 +21,7 @@
     "@fortawesome/free-brands-svg-icons": "^6.5.1",
     "@fortawesome/free-solid-svg-icons": "^6.5.1",
     "@fortawesome/react-fontawesome": "^0.2.0",
+    "@heroicons/react": "^2.0.11",
```
Comment (brunopacheco1) [inline package.json:24]: important: please use latest version: `2.1.5`

File: src/app/page.tsx [hunk 2]
```diff
@@ -52,25 +60,7 @@ const HomePage = () => {
...
-                d="M4.25 5.5a.75.75 0 00-.75.75v8.5c0 .414.336.75.75.75h8.5a.75.75 0 00.75-.75v-4a.75.75 0 011.5 0v4A2.25 2.25 0 0112.75 17h-8.5A2.25 2.25 0 012 14.75v-8.5A2.25 2.25 0 014.25 4h5a.75.75 0 010 1.5h-5z"
-                clipRule="evenodd"
-              ></path>
-              <path
-                fillRule="evenodd"
-                d="M6.194 12.753a.75.75 0 001.06.053L16.5 4.44v2.81a.75.75 0 001.5 0v-4.5a.75.75 0 00-.75-.75h-4.5a.75.75 0 000 1.5h2.553l-9.056 8.194a.75.75 0 00-.053 1.06z"
-                clipRule="evenodd"
-              ></path>
-            </svg>
+            Read more <ArrowTopRightOnSquareIcon className="w-5 h-5" />
```
Comment (brunopacheco1) [inline src/app/page.tsx:63]: important: please use `FontAwesomeIcon` instead, and remove new package.

**Comment by brunopacheco1** [comment]
suggestion: LNDC image is much better. We can keep it, if the license allows it.

**Pull request merged**